### DISCUSSION TITLE
Home Assistant - Vacuum Consumable Attributes not visible in Home Assistant

### DISCUSSION THREAD

**Discussion by piotrbielaska** [Hypfer/Valetudo#686] [Q&A / Support] [answered]
Hello. I have sucessufly installed newest Valetudo version on my Roborock S5 and connected it to Home Assistant via MQTT via autodiscovery. I can see my vacuum, but I do miss Consumable information. I see that valetudo is providing this information via MQTT, for example ConsumableStateAttribute_sensor_all

![image](https://user-images.githubusercontent.com/20399577/107791793-a2be6800-6d54-11eb-8418-55f91591d795.png)

Vacuum in Home Assistant doesn't have this information in Attributes:

![image](https://user-images.githubusercontent.com/20399577/107791971-df8a5f00-6d54-11eb-84fd-2f387c4f847e.png)

Any idea how to fix this? Should I manually add MQTT Vacuum in configuration.yaml?

**Comment by Hypfer** [comment, marked as answer]
The home assistant autodiscovery acts weirdly sometimes. It's not known yet what might be causing this

Rebooting HA, Valetudo or both often helps.
If not, try deleting the device entry in HA and repeat the reboot cycle.

It should appear eventually
